src/pages/designing/tutorials.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Tutorials
+description:
+  The Carbon design tutorials will get you setup to start designing with Carbon
```

Comment:
is setup supposed to match set up on line 13?

Reply:
Yeah, good catch! I just pushed a fix.